roles/rabbitmq/templates/etc/serverspec/rabbitmq_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'socket'
+
+files= ['shadow', 'passwd']
+files.each do |file|
+  describe file("/etc/#{file}") do
+    it { should contain 'rabbitmq'}
+  end
+end  
```

Comment:
Couldn't you use http://serverspec.org/resource_types.html#user to see if the user exists?